content/agent/_index.md

```diff
@@ -27,18 +33,114 @@ The Datadog Agent is software that runs on your hosts. It collects events and me
 
 {{< partial name="platforms/platforms.html" >}}
 
-The Agent has three main parts: the **collector**, **DogStatsD**, and the **forwarder**:
+## Agent Architecture
+
+{{< tabs >}}
+{{% tab "Agent v6" %}}
+
+Agent v6 is a complete rewrite in Go of the Agent v5. V6 offers better performances, smaller footprint, and more features. It is the default Datadog Agent (v5 is no longer in active development).
+
+Agent v6 is a composed of a main process responsible for collecting infrastructure metrics, logs, and receiving DogStatsD metrics. The main components to this process are:
+
+* The Collector is in charge of running checks and collecting metrics.
+* The Forwarder sends payloads to Datadog.
+
+Two optional processes are spawned by the Agent if enabled in the `datadog.yaml` configuration file:
+
+* The APM Agent is a process to collect traces (enabled by default).
+* The Process Agent is a process to collect live process information. By default, it only collects available containers, otherwise it is disabled.
+
+On Windows the services are listed as:
+
+| Service               | Description             |
+|-----------------------|-------------------------|
+| DatadogAgent          | “Datadog Agent”         |
+| datadog-trace-agent   | “Datadog Trace Agent”   |
+| datadog-process-agent | "Datadog Process Agent” |
+
+By default the Agent binds 3 ports on Linux and 4 on Windows and OSX:
+
+| Port | Description                                                                                 |
+|------|---------------------------------------------------------------------------------------------|
+| 5000 | Exposes runtime metrics about the Agent.                                                    |
+| 5001 | Used by the Agent CLI and GUI to send commands and pull information from the running Agent. |
+| 5002 | Used by the GUI server on Windows and OSX.                                                  |
+| 8125 | Used for the DogStatsD server to receive external metrics.                                  |
+
+### The Collector
+The collector gathers all standard metrics every 15 seconds. Agent v6 embed a Python2.7 interpreter to run integrations and [custom checks][1].
+
+### The Forwarder
+
+The Agent forwarder listens for incoming requests over HTTP to send metrics over HTTPS to Datadog. Buffering prevents network splits from affecting metric reporting. Metrics are buffered in memory until a limit in size or number of outstanding send requests are reached. Afterwards the oldest metrics are discarded to keep the forwarder's memory footprint manageable. Logs are sent over an SSL-encrypted TCP connection to Datadog.
```

Comment:
The forwarder don't listen at all. It takes payalod from the collector and send them to DD. The forwarder describe is the one in agent v5